### ISSUE TITLE
Add unittests with Jest

### ISSUE THREAD

**Issue by mve** [freave/make-pot#20]

**Comment by m10rten** [comment]
Do these unittests need to be in the same folder as the .php tests?

**Comment by mve** [comment]
> Do these unittests need to be in the same folder as the .php tests?

Yes i ended up deciding to put all the tests inside the "test" folder instead of alongside the code just to keep the src directory clean.

I've just committed a basic setup for Jest.
https://github.com/freave/make-pot/commit/6c6d67bb7a88d7dd02d094e0924f7fde36a455ad
